api/src/index.ts

```diff
@@ -3,10 +3,46 @@ import * as express from "express";
 const app = express();
 const port = 5000;
 
+// enabling 'http://localhost:4200' to use the api
+var cors = require('cors');
+var corsOptions = {
+  origin: 'http://localhost:4200',
+  optionsSuccessStatus: 200 
+};
+app.use(cors(corsOptions));
+
+// connect to DB
+const mongoose = require("mongoose");
+mongoose.connect("mongodb://localhost/auction-dev", {useNewUrlParser: true});
+const db = mongoose.connection;
+db.on('error', (error) => console.error(error));
+db.once('open', () => console.log('Connected to Database'));
+
+const Auction = require('./models/auction');
+
+// insert new object into DB
+// I don't know how to delete objects from previous sessions 😅
```

Comment:
Próbálj meg olyat, hogy nézd meg mentés előtt, hogy van-e ilyen nevű Auction. Ha van, akkor töröld (vagy ne töröld és ne is tegyél be az adatbázisba). Nem ellenőriztem le, de valami ilyesmi lehetne:
```
if (!Auction.exists({name: "Auction Name From MongoDB🔥"})) {
  first_auction.save();
}
```